jhub_apps/hub_client/hub_client.py

```diff
@@ -82,14 +87,14 @@ def start_server(self, username, servername):
         r.raise_for_status()
         return r.status_code, servername
 
-    def create_server(self, username, servername, user_options=None):
+    def create_server(self, username: str, servername: str, user_options: UserOptions = None):
```

Comment:
I understand that the API expansion need to be propagated to `routes.py` to actually be accessible from the frontend, but it is deferred to another PR?

Reply:
It propagates via `UserOptions`, which is updated here: https://github.com/nebari-dev/jhub-apps/pull/224/files#diff-cd4b38f2e6e6d470cb787c6a7e90d69b0a48e61bbaa026cc11cca116948d34ebR75

Reply:
I see, user options are passed via `ServerCreation`. Makes sense!